src/games/avalon/views/game/goOnMission.tsx

```diff
@@ -163,32 +164,39 @@ export const GoOnMission: React.SFC<{ mission: Mission }> = ({ mission }) => {
     return (
       <>
         <Typography gutterBottom variant="h2">
-          {!allCardsPlayed && 'Waiting for the others to play their cards'}
+          {!allCardsPlayed &&
+            <FullScreenNotice>
+              Waiting for the others to play their cards
+            </FullScreenNotice>
+          }
           {game.excalibur && !mission.hasSwitched && 'Waiting on excalibur to decide'}
-          {allCardsPlayed && (!game.excalibur || mission.hasSwitched) && 'Waiting on you to flip votes'}
+          {allCardsPlayed && (!game.excalibur || mission.hasSwitched) && 
+            <FullScreenNotice onClick={flipCards}>
```

Comment:
pass green color (make green a default color in the FullScreenNotice component) here to indicate that all cards were played and mission Leader/owner can flip cards to see the results